python/paddle/fluid/framework.py

```diff
@@ -2996,14 +2996,31 @@ def find_name(var_list, name):
                 for m in proto.outputs:
                     if (m.name not in outputs) and m.dispensable:
                         continue
-                    if not ((m.name in outputs) or m.dispensable):
-                        raise ValueError(
-                            (
-                                "Incorrect setting for output(s) of "
-                                "operator \"%s\", should set: [%s]."
+
+                    # Primitive mode currently doesn't support intermediate output
+                    if core._is_bwd_prim_enabled():
+                        if not (
+                            (m.name in outputs)
+                            or m.dispensable
```

Comment:
This branch will not stepped in?

Reply:
will step into this branch when the opdesc not contain intermediate output, but proto has intermediate output  define, such as xshape of reshap2